governance-exercises/DAO Masons poll-exercise-04.md

```diff
@@ -0,0 +1,58 @@
+# Poll 4
+
+### Determine what course of action you would take as a Governance Facilitator and communicate it clearly.
+
+Protocol Scope Facilitator Imogen has requested an urgent poll to gather MKR sentiment on if a costly security fix should go live.
+
+Our read on the situation:
+Fix B Executive is going up regardless, very shortly.
+Fix A Executive will be ready around the same time and may go up based on MKR holder sentiment.
+A Technical Ecosystem Actor and Protocol Scope Facilitator are requesting a poll to see if the DAO supports Fix A, which will likely result in a 20mm loss to MKR holders.
+
+I would start a chat with Geneveive and/or Imogen and verify the timeline, and ask them questions publicly on the forum as well. I'm going to pretend that they said the fixes would be ready in 8 hours.
+
+During this conversation I'm going to propose a different course of action rather than creating a poll about Fix A implementation. If they make an executive proposal for Fix A and Fix B, then they can go up at the same time and start their voting periods. If they both pass, then the DAO has spoken and both are executed. If only Fix B passes, then that's that and we are left vulnerable for 2 weeks.
+
+I could then spend the remaining time waiting for the execs to be ready 'coordinating an emergency governance call to discuss solutions and a plan for subsequent actions, explicitly inviting known stakeholders' as dictated by MIP24c3 [Emergency Response Procedure](https://mips.makerdao.com/mips/details/MIP24#MIP24c3).
```

Comment:
Good, though I would also point out that you need to provide executive copy for the two emergency executive votes.